content/quickstart/dotnet/tracing.md

```diff
@@ -0,0 +1,297 @@
+---
+title: "Tracing"
+date: 2019-01-07T12:01:21-07:00
```

Comment:
Please make this `date: 2019-01-07T00:01:21-07:00`

Reply:
done